cmd/branchingconfigmanagers/tide-config-manager/main.go

```diff
@@ -237,6 +237,136 @@ func (cfe codeFreezeEvent) GetDataFromProwConfig(pc *prowconfig.ProwConfig) {
 	}
 }
 
+type generalAvailabilityEvent struct {
+	repos                         sets.String
+	excludedAllowList             sets.String
+	noXYAllowList                 sets.String
+	openshiftReleaseBranches      sets.String
+	openshiftReleaseBranchesPlus1 sets.String
+	releasePast                   string
+	openshiftPast                 string
+	releaseCurrent                string
+	openshiftCurrent              string
+	releaseFuture                 string
+	openshiftFuture               string
+	past                          string
+	current                       string
+	future                        string
+}
+
+func newGeneralAvailabilityEvent(past, current, future string) generalAvailabilityEvent {
+	noXYAllowList := sets.NewString(
```

Comment:
That list should be controllable with flags or something outside the code.

Reply:
Yes, but the ticket that is going to figure out is in the backlog. It's still a prototype.

Reply:
I guess local config could be created for now, what do you think?

Reply:
A local config should be ok, and we can have it in release repo as well, so we can control its data.

Reply:
Done, extracted to a yaml config file. I am writing docs/howto file for that tool, but it will come in a separate PR (with an explanation of format etc). Generally, the config will look like this:
```
NoXYAllowList:
  - openshift/cluster-logging-operator
.....
ExcludedAllowList:
  - openshift/assisted-installer
....
```